docs/querying/select-in-depth.md

```diff
@@ -0,0 +1,900 @@
+is still relevant for other finder methods, as they are built on top of `findAll`.
+
+## Simple SELECT queries
+
+Without any options, [`findAll`](pathname:///api/v7/classes/_sequelize_core.index.Model.html#findAll) will return all rows of a table as instances of the model,
+and [`findOne`](pathname:///api/v7/classes/_sequelize_core.index.Model.html#findOne) will return the first row of a table as an instance of the model:
+
+```ts
+const users = await User.findAll();
+```
+
+```sql
+SELECT * FROM users;
+```
+
+## Selecting Attributes
+
+By default, querying methods will load all attributes of the model. You can control which attributes are loaded by using the [`attributes`](pathname:///api/v7/interfaces/_sequelize_core.index.Projectable.html) option:
+
+To select a specific list of attributes, set that option to an array of attribute names, like this:
+
+```js
+User.findAll({
+  // "firstName" and "lastName" are attributes of the "User" model
+  attributes: ['firstName', 'lastName'],
+});
+```
+
+This will result in roughly the following SQL query:
+
+```sql
+SELECT "firstName", "lastName" FROM "users";
+```
+
+### Excluding attributes
+
+You also have the option to exclude attributes from the result set:
+
+```js
+User.findAll({
+  // "password" is attributes of the "User" model
+  attributes: { exclude: ['password'] },
+});
+```
+
+This will load all attributes that have been defined in the model, except for the `password` attribute.
+
+### Extra attributes
+
+You also have the option to load custom attributes that are not part of the model definition. 
+This is useful if you need to compute the value of one of your attributes, or if you need to load attributes that are not part of your model definition.
+
+You can do this by using the `attributes.include` option, which will load all attributes that have been defined in the model, plus the extra attributes that you have specified:
+
+```js
+User.findAll({
+  attributes: {
+    include: [
+      // This will include a dynamically computed "age" property on all returned instances.
+      [sql`DATEDIFF(year, "birthdate", GETDATE())`, 'age'],
+    ],
+  },
+});
+```
+
+You can use [SQL Literals](./raw-queries.md#literals--raw-sql-) to select any SQL expression instead of a column.
+When using SQL expressions like in the above example, you must give it an alias to be able to access it from the model.
+In the above example, the alias is `age`.
+
+:::caution TypeScript
+
+Be aware that these attributes will not be typed, as methods such as `findAll` and `findOne` return 
+instances of the model class.
+
+If these attributes are part of your model, you could declare them as optional attributes on your model.
+
+If they are not part of your model, 
+One way to type these attributes is to use the `raw` option, which will return a plain object instead of an instance of the model class:
```

Comment:
```suggestion
If they are not part of your model; 
one way to type these attributes is to use the `raw` option, which will return a plain object instead of an instance of the model class:
```